include/swift/SILOptimizer/PassManager/PassPipeline.h

```diff
@@ -56,7 +56,9 @@ class SILPassPipelinePlan final {
   SILPassPipelinePlan() = default;
   ~SILPassPipelinePlan() = default;
   SILPassPipelinePlan(const SILPassPipelinePlan &) = default;
+#if !defined(_MSC_VER) || defined(__clang__)
   SILPassPipelinePlan(SILPassPipelinePlan &&) = delete;
```

Comment:
I'm not sure that's the right thing to do.

The errors are
> 1>C:\Users\hbellamy\Documents\GitHub\my-swift\swift\lib\SILOptimizer\PassManager\PassPipeline.cpp(104): error C2280: 'swift::SILPassPipelinePlan::SILPassPipelinePlan(swift::SILPassPipelinePlan &&)': attempting to reference a deleted function
> 1>  C:\Users\hbellamy\Documents\GitHub\my-swift\swift\include\swift/SILOptimizer/PassManager/PassPipeline.h(51): note: see declaration of 'swift::SILPassPipelinePlan::SILPassPipelinePlan'

It comes from [this code](https://github.com/apple/swift/blob/master/lib/SILOptimizer/PassManager/PassPipeline.cpp)

```
SILPassPipelinePlan
SILPassPipelinePlan::getDiagnosticPassPipeline(SILOptions Options) {
  SILPassPipelinePlan P;

  if (SILViewSILGenCFG) {
    addCFGPrinterPipeline(P, "SIL View SILGen CFG");
  }

  // If we are asked do debug serialization, instead of running all diagnostic
  // passes, just run mandatory inlining with dead transparent function cleanup
  // disabled.
  if (Options.DebugSerialization) {
    addMandatoryDebugSerialization(P);
    return P;
  }

  // Lower all ownership instructions right after SILGen for now.
  addOwnershipModelEliminatorPipeline(P);

  // Otherwise run the rest of diagnostics.
  addMandatoryOptPipeline(P);

  if (SILViewGuaranteedCFG) {
    addCFGPrinterPipeline(P, "SIL View Guaranteed CFG");
  }
  return P;
}
```